Update add-clustervirtualmachinerole.md (#1197)
MicrosoftDocs/windows-powershell-docs: docset/windows/failoverclusters/add-clustervirtualmachinerole.md

Instead, if that server, or cluster node, fails or requires scheduled maintenance, then another node begins to run the virtual machines instead through a process known as failover.
The virtual hard disk (VHD) file for the clustered virtual machine must be on the clustered disk used by that virtual machine.

> > [!NOTE]This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.

Reviewer: ```suggestion
> [!NOTE] This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
```

> > [!NOTE]This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For more information, please go to: https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344

Reviewer: ```suggestion
> [!NOTE] This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For more information, please go to: https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344
```

Instead, if that server, or cluster node, fails or requires scheduled maintenance, then another node begins to run the virtual machines instead through a process known as failover.
The virtual hard disk (VHD) file for the clustered virtual machine must be on the clustered disk used by that virtual machine.

> 

Reviewer 2: ```suggestion
```

> > **Note:** This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer. Also this cmdlet generate a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet otherwise you will receive an exception "The File Exist".

Reviewer: ```suggestion
> [!Note]
> This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.

> [!Note]
> This cmdlet generates a .TMP file under the user temp folder, and you can't have more than 65535 .TMP files generated by this cmdlet, otherwise you will receive an exception "The File Exist". For more information, please go to:
https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344
```

Reviewer 2: ```suggestion
>  [!NOTE}
> This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer. Also, this cmdlet generate a .TMP file in the user temp folder, and if more than 65535 .TMP files are generated by this cmdlet you will get the error: "The File Exists". For more information, see this [support article](https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344).
```

> https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344

Reviewer: ```suggestion
```

> https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/TQA-Add-ClusterVirtualMachineRole-fails-with-the-error-quot-The/ba-p/713344

Reviewer: ```suggestion
```

> 

Reviewer 2: ```suggestion
```

> 

Reviewer 2: ```suggestion
```
